src/ol/format/wmtscapabilitiesformat.js

```diff
@@ -353,6 +367,23 @@ ol.format.WMTSCapabilities.TMS_LINKS_PARSERS_ = ol.xml.makeParsersNS(
  * @type {Object.<string, Object.<string, ol.xml.Parser>>}
  * @private
  */
+ol.format.WMTSCapabilities.DIMENSION_PARSERS_ = ol.xml.makeParsersNS(
+    ol.format.WMTSCapabilities.NAMESPACE_URIS_, {
+      'Default': ol.xml.makeObjectPropertySetter(
+          ol.format.XSD.readString),
+      'Value': ol.xml.makeObjectPropertyPusher(
+          ol.format.XSD.readString)
+    }, ol.xml.makeParsersNS(ol.format.WMTSCapabilities.OWS_NAMESPACE_URIS_, {
+      'Identifier': ol.xml.makeObjectPropertySetter(
```

Comment:
I see various places where Identifier is parsed in src/ol/format/wmtscapabilitiesformat.js but none of them uses something from owsformat.js.

I tried to add a ol.format.OWS.readIdentifier in owsformat but I always get undefined. I fail to see how to do that.
- in owsformat:
```js
/**
 * @param {Node} node Node.
 * @param {Array.<*>} objectStack Object stack.
 * @returns {Object|undefined}
 */
ol.format.OWS.readIdentifier = function(node, objectStack) {
  return ol.xml.pushParseAndPop({}, ol.format.OWS.IDENTIFIER_PARSERS_,
      node, objectStack);
};


/**
 * @const
 * @type {Object.<string, Object.<string, ol.xml.Parser>>}
 * @private
 */
ol.format.OWS.IDENTIFIER_PARSERS_ = ol.xml.makeParsersNS(
    ol.format.OWS.NAMESPACE_URIS_, {
      'Identifier': ol.xml.makeObjectPropertySetter(
          ol.format.XSD.readString)
    });
```
- in wmtscapabilitiesformat.js:
```js
ol.format.WMTSCapabilities.DIMENSION_PARSERS_ = ol.xml.makeParsersNS(
    ol.format.WMTSCapabilities.NAMESPACE_URIS_, {
      'Default': ol.xml.makeObjectPropertySetter(
          ol.format.XSD.readString),
      'Value': ol.xml.makeObjectPropertyPusher(
          ol.format.XSD.readString)
    }, ol.format.WMTSCapabilities.OWS_NAMESPACE_URIS_, {
      'Identifier':  ol.xml.makeObjectPropertySetter(
          ol.format.OWS.readIdentifier)
    });
```